### DISCUSSION TITLE
ICU4X 1.3 (Oct 4, 2023) - assets not up to date?

### DISCUSSION THREAD

**Discussion by jlfaucher** [unicode-org/icu4x#4125] [Q&A] [answered]
Hello

I get this message when clicking on the tag icu@1.3.2 in the Releases view.
"This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

In both assets, the file CHANGELOG.md starts with
"icu4x 1.2.x (Apr 17, 2023)"

Is there a problem with this release?

**Comment by robertbastian** [comment, marked as answer]
The tagged 1.3.2 is indeed not on the main branch (our only branch), as we had to do some cherry-picks. I don't think that that should be a problem.

The changelog only got updated after tagging, so it does not show 1.3 yet, our bad. If you look at the version in the top-level `Cargo.toml`, it is 1.3.2.

**Reply by sffc** [reply]
Can we change the GitHub 1.3 release to instead be on https://github.com/unicode-org/icu4x/commit/3670b3c7b6a0b77ce6514a78ecae1cf6e0931f0a

**Reply by robertbastian** [reply]
Releases are on tags, and `1.3.2` is taken now. https://github.com/unicode-org/icu4x/commit/3670b3c7b6a0b77ce6514a78ecae1cf6e0931f0a also contains changes that are not in 1.3.2, so I think that could be confusing. We could add another commit on the 1.3.2 branch that contains the changelog, but it would need a fresh tag.

**Reply by Manishearth** [reply]
In general I am okay with the release tags not containing the updated changelog.


But we should probably start crafting the changelog in advance so we can be better about this.
